src/app/shell/item-comparators.ts

```diff
@@ -150,7 +150,21 @@ const ITEM_COMPARATORS: {
   // new -> old
   acquisitionRecency: acquisitionRecencyComparator,
   // None -> Kinetic -> Arc -> Thermal -> Void -> Raid -> Stasis
-  element: compareBy((item) => item.element?.enumValue ?? Number.MAX_SAFE_INTEGER),
+  elementWeapon: compareBy((item) => {
+    if (item.itemCategoryHashes.includes(ItemCategoryHashes.Weapon)) {
+      return item.element?.enumValue ?? Number.MAX_SAFE_INTEGER;
+    } else {
```

Comment:
can save some code and eliminate these else blocks. when a function hits its final `}` without a return statement, it returns `undefined` on its own